config/routes.rb

```diff
@@ -202,6 +202,8 @@
 
     resources :pre_auth_states, only: [:index]
 
+    resources :banners
```

Comment:
You don't have a show action in the controller, so would be good to use `except: :show` here.

Reply:
Oh yeah, I missed that.